docs/detections/prebuilt-rules/rule-details/bypass-uac-via-event-viewer.asciidoc

```diff
@@ -68,6 +68,9 @@ process.parent.name : "eventvwr.exe" and not process.executable :
 [[bypass-uac-via-event-viewer-history]]
 ==== Rule version history
 
+Version 9 (7.15.0 release)::
```

Comment:
This might be an extra colon, but it doesn't look to be affecting the HTML output, so maybe it's not an issue?